torchqdynamics/mesolve.py

```diff
@@ -1,27 +1,90 @@
-from typing import Any, Dict, List, Optional
+from typing import List, Tuple
 
 import numpy as np
 import torch
-import torchdiffeq
+from tqdm import tqdm
+
+from torchqdynamics import hermitian
 
 
 def mesolve(
     H: torch.Tensor,
-    Ls: List[torch.Tensor],
     rho0: torch.Tensor,
-    tlist: np.ndarray,
-    solver: str = 'rk4',
-    options: Optional[Dict[str, Any]] = None,
-) -> torch.Tensor:
-    if options is None:
-        options = {}
+    times: np.ndarray,
+    c_ops: List[torch.Tensor] = None,
```

Comment:
Could we pick `jump_ops` rather? I know it's different from QuTiP, but naming-wise QuTiP has some very bad standard so we should be careful.